### PR TITLE
Added localization for wooden planks, fence, tree, etc.

### PR DESCRIPTION
**Pull request by randomei**

### CODE REVIEW

File: locale/moretrees.de.tr [hunk 1]
```diff
@@ -209,3 +209,12 @@ Willow Tree Trunk=Weidenstamm
 Willow Tree Trunk Slab=Weidenstammplatte
 Willow Tree Trunk Stair=Weidenstammstufe
 Yellow=gelb
+
+# init.lua
+Sapling=
```
Comment (SwissalpS) [inline locale/moretrees.de.tr:214]: Setzling
Baum
(Holz)bretter
Blätter
Holzzaun
Holzschiene
Holztor

File: locale/moretrees.es.tr [hunk 2]
```diff
@@ -209,3 +209,12 @@ Willow Tree Trunk=Tronco de Árbol de Sauce
 Willow Tree Trunk Slab=Losa de Tronco de Árbol de Sauce
 Willow Tree Trunk Stair=Escalera de Tronco de Árbol de Sauce
 Yellow=Amarillo
+
+# init.lua
+Sapling=
```
Comment (SwissalpS) [inline locale/moretrees.es.tr:214]: Retoño
Tablones
Hojas
Valla de Madera
Riel de Valla( de Madera)
Puerta de Valla( de Madera)

File: locale/moretrees.fr.tr [hunk 3]
```diff
@@ -209,3 +209,12 @@ Willow Tree Trunk=Tronc d'arbre de saule
 Willow Tree Trunk Slab=Dalle en tronc de saule
 Willow Tree Trunk Stair=Escalier en tronc de saule
 Yellow=jaune
+
+# init.lua
+Sapling=
```
Comment (SwissalpS) [inline locale/moretrees.fr.tr:214]: Jeune Arbre / Pousse d'arbre
Arbre
Planches( de bois)
Feuilles
Barrière en bois
Clôture en bois
Porte de clôture en bois

File: locale/moretrees.es.tr (hunk 2 above)
Comment (SwissalpS) [reply locale/moretrees.es.tr:214]: I'm not sure it is necessary to add 'de madera' to wooden parts. Fences made of special wood or other materials can add that, but for default wood I don't see the need.
In the French and German translations it makes more sense, IMO.

File: locale/moretrees.fr.tr (hunk 3 above)
Comment (SwissalpS) [reply locale/moretrees.fr.tr:214]: 'Planches' doesn't need the 'de bois' part when not specifying what type of wood. It's not wrong to add it. 'Planches' are generally made of wood.

**Comment by randomei** [comment]
@SwissalpS thank you for your suggestions! I've added this strings. Also added russian locale.

File: locale/moretrees.es.tr [hunk 4]
```diff
@@ -211,10 +211,10 @@ Willow Tree Trunk Stair=Escalera de Tronco de Árbol de Sauce
...
 # init.lua
-Sapling=
-Tree=
-Wooden Planks=
-Leaves=
-Wooden Fence=
-Wooden Fence Rail=
-Wooden Fence Gate=
+Sapling=Retoño
+Tree=Puu
```
Comment (SwissalpS) [inline locale/moretrees.es.tr:215]: I have never heard of "Puu" and neither does RAE know about this. "Árbol" is the general word for tree.

File: locale/moretrees.es.tr [hunk 5]
```diff
@@ -211,10 +211,10 @@ Willow Tree Trunk Stair=Escalera de Tronco de Árbol de Sauce
...
+Wooden Planks=Tablones
+Leaves=Hojas
+Wooden Fence=Riel de Valla
```
Comment (SwissalpS) [inline locale/moretrees.es.tr:218]: Wooden Fence=Valla
or
Valla de Madera
if you want to specify that it is wooden.
An alternative word would be "cerca" but I think "Valla" is a better fit.

Comment (randomei) [reply locale/moretrees.es.tr:218]: Ok

File: locale/moretrees.fr.tr [hunk 6]
```diff
@@ -211,10 +211,10 @@ Willow Tree Trunk Stair=Escalier en tronc de saule
...
-Leaves=
-Wooden Fence=
-Wooden Fence Rail=
-Wooden Fence Gate=
+Sapling=Jeune Arbre
+Tree=Arbre
+Wooden Planks=Planches
+Leaves=Feuilles
+Wooden Fence=Barrière en bois
+Wooden Fence Rail=Clôture en bois
```
Comment (SwissalpS) [inline locale/moretrees.fr.tr:219]: Clôture is a synonym of barrière. Probably the better word for "Fence".
So, "Wooden Fence Rail" should be "Rail de clôture en bois"

**Comment by SwissalpS** [comment]
LGTM, my Russian is too restricted to say anything about that ;)

**Comment by zmv7** [comment]
LGTM

**Review by wsor4035 — APPROVED** [review]
code seems fine, french handled, russian approved over on mtd/ltd (whatever we call it now?)

**Pull request merged**